### ISSUE TITLE
org.jetbrains.kotlin.psi.KtStringTemplateExpression

### ISSUE THREAD

**Issue by vlad20012** [intellij-rust/intellij-rust#1476]
```
org.jetbrains.kotlin.psi.KtStringTemplateExpression cannot be cast to org.rust.lang.core.psi.ext.RsCompositeElement
java.lang.ClassCastException: org.jetbrains.kotlin.psi.KtStringTemplateExpression cannot be cast to org.rust.lang.core.psi.ext.RsCompositeElement
	at org.rust.lang.core.resolve.NameResolutionKt.walkUp(NameResolution.kt:755)
	at org.rust.lang.core.resolve.NameResolutionKt.processMacroCallVariants(NameResolution.kt:318)
	at org.rust.lang.core.resolve.ref.RsMacroCallReferenceImpl$resolveInner$1.invoke(RsMacroCallReferenceImpl.kt:22)
	at org.rust.lang.core.resolve.ref.RsMacroCallReferenceImpl$resolveInner$1.invoke(RsMacroCallReferenceImpl.kt:16)
	at org.rust.lang.core.resolve.ProcessorsKt.collectResolveVariants(Processors.kt:51)
	at org.rust.lang.core.resolve.ref.RsMacroCallReferenceImpl.resolveInner(RsMacroCallReferenceImpl.kt:22)
	at org.rust.lang.core.resolve.ref.RsReferenceBase$Resolver.resolve(RsReferenceBase.kt:68)
	at org.rust.lang.core.resolve.ref.RsReferenceBase$Resolver.resolve(RsReferenceBase.kt:66)
	at com.intellij.psi.impl.source.resolve.ResolveCache.lambda$resolve$0(ResolveCache.java:140)
	at com.intellij.openapi.util.RecursionManager$2.doPreventingRecursion(RecursionManager.java:112)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolve(ResolveCache.java:139)
	at com.intellij.psi.impl.source.resolve.ResolveCache.resolveWithCaching(ResolveCache.java:205)
	at org.rust.lang.core.resolve.ref.RsReferenceBase.advancedCachedMultiResolve(RsReferenceBase.kt:42)
	at org.rust.lang.core.resolve.ref.RsReferenceBase.multiResolve(RsReferenceBase.kt:35)
	at com.intellij.psi.PsiPolyVariantReferenceBase.resolve(PsiPolyVariantReferenceBase.java:46)
	at org.rust.lang.core.resolve.ref.RsReferenceBase.resolve(RsReferenceBase.kt:29)
	at org.rust.lang.core.resolve.ref.RsReferenceBase.resolve(RsReferenceBase.kt:22)
	at com.intellij.codeInsight.TargetElementUtil.doGetReferenceOrReferencedElement(TargetElementUtil.java:391)
	at com.intellij.codeInsight.TargetElementUtil.getReferenceOrReferencedElement(TargetElementUtil.java:368)
	at com.intellij.codeInsight.TargetElementUtil.doFindTargetElement(TargetElementUtil.java:238)
	at com.intellij.codeInsight.TargetElementUtil.findTargetElement(TargetElementUtil.java:210)
	at com.intellij.codeInsight.daemon.impl.IdentifierHighlighterPass.doCollectInformation(IdentifierHighlighterPass.java:102)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:438)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1153)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:431)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:556)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:501)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:430)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:406)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:142)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:242)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:404)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:165)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```

**Comment by farodin91** [comment]
Could solved by my fix #1475

**Comment by farodin91** [comment]
#1479
